### ISSUE TITLE
Add countdown for next game

### ISSUE THREAD

**Issue by passsy** [hitherejoe/FootballFace#1]
It's a watch face. So adding time related things could be a good idea. Why not adding a countdown until the next game of my team starts. Maybe hide it behind a click on the watch face to keep the beauty?
Has nfl.com a good api? is using the non public aip of nfl gamepass ok for this project ok?

**Comment by hitherejoe** [comment]
Yes that's great! I had a few ideas of things to add and this was one of them. I know NFL have a fantasy data API, but I'm happy use an unofficial one if there is one. I found an API for realtime scores etc (https://profootballapi.com), so something interesting could be done with that in future...
